src/integrationTest/java/net/sf/jabref/gui/EntryTableTest.java

```diff
@@ -0,0 +1,91 @@
+package net.sf.jabref.gui;
+
+import java.awt.event.KeyEvent;
+import java.io.File;
+import java.util.regex.Pattern;
+
+import net.sf.jabref.JabRefMain;
+
+import org.assertj.swing.fixture.FrameFixture;
+import org.assertj.swing.fixture.JFileChooserFixture;
+import org.assertj.swing.fixture.JTableCellFixture;
+import org.assertj.swing.fixture.JTableFixture;
+import org.assertj.swing.junit.testcase.AssertJSwingJUnitTestCase;
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.assertj.swing.finder.WindowFinder.findFrame;
+import static org.assertj.swing.launcher.ApplicationLauncher.application;
+
+
+/**
+ * Specific Use-Case:
+ * I import a database. Then I doubleclick on the first entry in the table to open the entry editor.
+ * Then I click on the first entry again, and scroll through all of the lists entries, without having to click
+ * on the table again.
+ */
+public class EntryTableTest extends AssertJSwingJUnitTestCase {
+
+    private final static int SCROLL_ACTION_EXECUTION = 5;
+    private final static String TEST_FILE_NAME = "testjabref.bib";
+
+    private int down = KeyEvent.VK_DOWN;
+
+    @Override
+    protected void onSetUp() {
+        AWTExceptionHandler awtExceptionHandler = new AWTExceptionHandler();
+        awtExceptionHandler.installExceptionDetectionInEDT();
+        application(JabRefMain.class).start();
+
+
+        robot().settings().timeoutToFindSubMenu(1_000);
+        robot().settings().delayBetweenEvents(50);
+    }
+
+    @Test
+    public void scrollThroughEntryList() throws InterruptedException {
+        FrameFixture mainFrame = findFrame(JabRefFrame.class).withTimeout(10_000).using(robot());
+        robot().waitForIdle();
+
+        mainFrame.menuItemWithPath("File", "Import into new database").click();
+        JFileChooserFixture openFileChooser = mainFrame.fileChooser();
+
+        //reduce the delay to speed up loading the test resource
+        robot().settings().delayBetweenEvents(10);
+        openFileChooser.fileNameTextBox().enterText(getTestFilePath(TEST_FILE_NAME));
+        openFileChooser.approveButton().click();
+        robot().settings().delayBetweenEvents(50);
+
+        JTableFixture entryTable = mainFrame.table();
+
+        Pattern pattern = Pattern.compile("256.*");
+        JTableCellFixture firstCell = entryTable.cell(pattern);
```

Comment:
would be nice if you would found out a way to always select the first cell and that it not depends on the content.